### PR TITLE
Feature: add giv power to user by address query

### PR DESCRIPTION
**Pull request by CarlosQ96**
Related to https://github.com/Giveth/impact-graph/issues/718#event-7780007932

For frontends. Please update to this query (with givPower and boostedProjectsCount)
```
query ($address: String!) {
    userByAddress(address: $address) {
      id
      firstName
      lastName
      name
      email
      avatar
      walletAddress
      url
      location
      isSignedIn
      boostedProjectsCount
      givPower
      likedProjectsCount
      donationsCount
      projectsCount
    }
  }
```

### CODE REVIEW

**Comment by CarlosQ96** [comment]
@MohammadPCh once this is merged. You add to the queried field this two:
boostedProjectsCount and givPower,  both integers, it's either a a positive number or zero always.

**Comment by CarlosQ96** [comment]
@mohammadranjbarz Let me know if I have to change the query, I tried to understand how the tables work of powerboosting. Hopefully I understood correctly.

File: src/repositories/powerBalanceSnapshotRepository.ts [hunk 1]
```diff
@@ -10,3 +10,16 @@ export const createPowerSnapshotBalances = async (
...
   await PowerBalanceSnapshot.save(powerBalanceSnapshots);
 };
+
+export const findPowerBalanceByUserId = async (
+  userId: number,
+): Promise<number> => {
+  const balance = await PowerBalanceSnapshot.createQueryBuilder('powerBalance')
+    .select('powerBalance.balance AS "givPower"')
+    .where('powerBalance.userId = :userId', { userId })
+    .orderBy('powerBalance.id', 'DESC')
```
Comment (CarlosQ96) [inline src/repositories/powerBalanceSnapshotRepository.ts:20]: This is to always grab the latest snapshot

**Comment by mohammadranjbarz** [comment]
@CarlosQ96  I just talked with Cherik it seems they need to have current givPower balance of users not balance in , so they have to get it directly from subgraph, and `boostedProjectsCount` is just enough for that we already have it, so unfortunately I guess we dont't need to have implement this anymore, 
@MohammadPCh  Am I right?

**Comment by CarlosQ96** [comment]
Ok let me know and i'll close down this PR!

**Pull request closed without merging**